### DISCUSSION TITLE
Is there a way to change the theme? or change the interface?

### DISCUSSION THREAD

**Discussion by bujary** [winder/Universal-G-Code-Sender#2002] [Help]
Is there a way to change the theme? or change the interface?

**Comment by breiler** [comment]
Reorganizing the UI: https://github.com/winder/Universal-G-Code-Sender/wiki/Configuration#configuring-panels-and-windows
Changing the theme (Look and feel): https://github.com/winder/Universal-G-Code-Sender/wiki/Configuration#using-dark-theme

**Reply by MadTooler** [reply]
Is there a wiki or notes on creating a new "look and feel" theme?
Thanks

**Reply by breiler** [reply]
We don't have any specific instructions for creating new themes. 
Google should be your friend: `netbeans creating look and feel theme`

**Reply by MadTooler** [reply]
Thanks.  I will look into it.
